lib/ts/normalisedURLDomain.ts

```diff
@@ -17,14 +17,24 @@ import { URL } from "url";
 import { isAnIpAddress } from "./utils";
 
 export default class NormalisedURLDomain {
```

Comment:
this class should not be changed. It's purpose is only to take a string and normalise it.